### ISSUE TITLE
Potential loss in accuracy from using the compressed 8-bit dataset

### ISSUE THREAD

**Issue by RyanAmaudruz** [zhu-xlab/SSL4EO-S12#18]
Firstly, thank you for adding so much detail in your paper and repository. I think your work will greatly help progress SSL research on remote sensing data.
I notice that the code in "ssl4eo_dataset.py" loads the uncompressed data, resizes the patches to 264x264 before saving the patches in 8-bit. I imagine that this is the dataset that was tested when you report the accuracy with different precisions in Appendix 3A.
The compressed dataset however has not yet been resized and is already in 8-bit. Am I correct to assume that if one resizes the patches from the compressed dataset, there would be a further loss in accuracy, which might impact the SSL outcome beyond what was reported in Appendix 3A?

**Comment by moonboy12138** [comment]
@RyanAmaudruz 
  Hello, I would like to inquire whether the performance of your model has not decreased after pre-training with compressed uint8 datasets? I am experiencing difficulty achieving the accuracy mentioned in the paper when attempting pre-training with Masked Autoencoders(MAE) on the uint8 dataset.

**Comment by RyanAmaudruz** [comment]
Currently, I have not been able to repeat the results. I tried with the S2A dataset, while the authors use the S2C dataset.
I will try with the S2C dataset and let you know if I am able to reproduce the results.
In the meantime, good luck and please let me know if you have success.
Cheers

**Comment by wangyi111** [comment]
We've got other similar feedbacks on the compressed dataset, seems the information loss does result in some unignorable performance drop. We plan to update a lossless version. We've also updated a faster access to the raw data via [HuggingFace](https://huggingface.co/datasets/wangyi111/SSL4EO-S12/tree/main/s2_l1c). Looking forward to your further feedbacks as well!

**Comment by RyanAmaudruz** [comment]
@wangyi111 Would you have the log for the Dino s2c pretrain run by any chance?
That would be very helpful to see if my pretraining tracks with yours.
Again, thanks for your answers. They have been very helpful:)

**Comment by wangyi111** [comment]
Hi @RyanAmaudruz Yes the logs for the pretraining are available in the readme file already: "logs" entry as shown in the screenshot below:

<img width="551" alt="image" src="https://github.com/zhu-xlab/SSL4EO-S12/assets/35645723/510fc1a8-aa0f-41d8-9cd4-0678ab6431ce">
